__tests__/articles.test.js

```diff
@@ -28,8 +28,25 @@ describe("GET /api/articles/:id ", () => {
           created_at: expect.any(String),
           votes: expect.any(Number)
         }))
-        });
       });
+  });
+  it("status 200: returns the article with a comment_count, if there are any", () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({body}) => {
+        expect(body.article).toEqual(expect.objectContaining({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          comment_count: expect.any(String)
```

Comment:
We also need to prove that the comment_count is correct, not only that it is a string